src/components/Filters/Switch.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { Row, Switch as AntSwitch } from 'antd';
+import { styles } from '../../utils/constants';
+
+class Switch extends Component {
```

Comment:
Since there is only one prop `handleChange` and we know it doesn't change, we should use a PureComponent instead.